webui/react/src/utils/analytics.ts

```diff
@@ -0,0 +1,75 @@
+import { DeterminedInfo } from 'types';
```

Comment:
suggestion: I think this file should be outside `utils/` as it's less  of a utility and more of self-contained module giving us analytics

Reply:
great point, will move it out